src/app/components/itinerary-form/itinerary-form.component.html

```diff
@@ -82,14 +83,14 @@ <h3 class="text-lg font-bold">Create a Tag!</h3>
       <div class="mb-4 flex flex-col">
         <button
           type="button"
-          class="btn mb-2 w-fit self-center bg-white"
+          class="mb-2 w-fit self-center rounded-t-lg border-b-[1px] border-white bg-slate-1000 p-4 text-white hover:bg-transparent"
```

Comment:
You could just use ```border-b``` to set the bottom border to 1px.

Reply:
Thank you 🙏🏻